src/components/o-policy-features/_preview.html

```diff
@@ -1,27 +1,27 @@
 <axa-policy-features title="A 5 star car insurance with affordable premium services">
 
   <axa-policy-feature-item title="Get Discount"
-                           src="https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/beacon.svg"
                            alt="Discount Svg Icon"
                            description="We reward safe drivers : 75% no claims discount + an extra 10% off if you get a quote online">
+    <svg class="js-m-policy-features-icon m-policy-feature-item__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 96"><g fill="currentColor"><path d="M22.79 58.42a4.92 4.92 0 1 0 4.86 4.92 4.89 4.89 0 0 0-4.86-4.92zm0 7.84a2.92 2.92 0 1 1 2.86-2.92 2.89 2.89 0 0 1-2.86 2.92zm51.12-7.84a4.92 4.92 0 1 0 4.86 4.92 4.89 4.89 0 0 0-4.86-4.92zm0 7.84a2.92 2.92 0 1 1 2.86-2.92 2.89 2.89 0 0 1-2.86 2.92z"></path><path d="M92.42 48.84c-.35-3-2.44-6.4-5.58-7.16l-16-4.82c-1.47-1.28-11.63-10-17.06-12.52S38 23.47 29.1 25l-2.26.37a1 1 0 0 0-.37.14c-1.07.68-2.57 1.83-4.3 3.17-4.62 3.57-11.6 9-15.64 9a1 1 0 0 0-.34.07h-.18a1 1 0 0 0-.15.12 1 1 0 0 0-.19.28v.08a1 1 0 0 0-.07.35v6.74a3.54 3.54 0 0 0-2.6 3.4v11.93a4 4 0 0 0 2.52 3.73l4.38 1.77a1 1 0 0 0 .37.07h3.44a9.46 9.46 0 0 0 18.14 0h33a9.47 9.47 0 0 0 18.16 0 1 1 0 0 0 .26.05h7.09a4.66 4.66 0 0 0 4.64-4.7V53.1c0-2.32-.95-3.85-2.58-4.26zm-2.05-.19c-3.44-.46-5.17-3.07-6-5.63l2 .6c2.04.49 3.63 2.84 3.99 5.02zm-79.86-3.46h-3v-4.73a7.84 7.84 0 0 1 3 4.73zm-4 2h5.06a1 1 0 0 0 1-1 9.25 9.25 0 0 0-3.5-7c4.47-1.35 10-5.65 14.3-8.94 1.6-1.23 3-2.31 4-3l2.06-.34c8.24-1.37 18.49-3.07 23.49-.78 5.47 2.51 16.63 12.25 16.74 12.35a1 1 0 0 0 .37.21l12 3.62c1.35 5.39 4.69 8.35 9.41 8.35.42 0 1.53 0 1.53 2.38v5.75H82.31a9.43 9.43 0 0 0-16.81 0H43V45h24.78a1 1 0 0 0 .68-1.74A161.15 161.15 0 0 0 51.55 30l-.17-.09c-2.85-1.2-9.77-.88-20.55.94a1 1 0 0 0-.54.28c-.84.84-1.86 1.86-2.94 3l-.11.12c-2.75 2.8-6 6.18-8.31 9.13a1 1 0 0 0 .79 1.62H41v13.85h-9.81a9.43 9.43 0 0 0-16.81 0H5V48.72a1.53 1.53 0 0 1 1.53-1.53zM21.84 43c1.59-1.86 3.41-3.82 5.16-5.64V43zM29 35.28l2.48-2.5c3.89-.65 7-1.05 9.52-1.29V43H29zm14-4c4.56-.3 6.61.05 7.53.41A156.85 156.85 0 0 1 65.14 43H43zM10.47 64.22l-4.21-1.7A2 2 0 0 1 5 60.85h8.59a9.71 9.71 0 0 0-.33 2.49v.89zM22.79 71a7.64 7.64 0 1 1 7.5-7.64 7.58 7.58 0 0 1-7.5 7.64zm9.46-6.77v-.86a9.71 9.71 0 0 0-.25-2.52h32.74a9.71 9.71 0 0 0-.33 2.49v.86zM73.91 71a7.64 7.64 0 1 1 7.5-7.64 7.58 7.58 0 0 1-7.5 7.64zm16.44-6.75h-7v-.89a9.71 9.71 0 0 0-.33-2.49H93v.72a2.66 2.66 0 0 1-2.65 2.63z"></path></g></svg>
```

Comment:
Guess this is a requirement of how icons are implemented in AEM?

Reply:
this allows to pass svg from AEM as in similar components with the full SVG source, SVG is stored in DAM.
Having an svg rather than <img> tag allows to color the svg correctly